pkg/authorization/authorizer/scope/converter.go

```diff
@@ -122,6 +123,15 @@ func (userEvaluator) ResolveRules(scope, namespace string, clusterPolicyGetter r
 	}
 }
 
+// escalatingScopeResources are resources that are considered escalating for scope evaluation
+var escalatingScopeResources = []unversioned.GroupResource{
+	{Group: kapi.GroupName, Resource: "secrets"},
+	/*imageapi.GroupName*/ {Group: "", Resource: "imagestreams/secrets"},
```

Comment:
> I don't really understand why these are comments instead of the actual group names

We don't want to introduce a dependency from this package to that one.  Authorization should be a clean-ish layer, if possible.